gnat2goto/driver/tree_walk.adb

```diff
@@ -335,6 +335,33 @@ package body Tree_Walk is
 
    procedure Remove_Entity_Substitution (E : Entity_Id);
 
+   function Create_Dummy_Irep return Irep;
```

Comment:
Note: Not a new function, this is just moving an existing function declaration upwards

Reply:
Yes, I needed to see the declaration in the new reporting functions. I guess the commit description is misleading.